include/dts-functions.sh

```diff
@@ -338,9 +338,12 @@ board_config() {
           case $BOARD_MODEL in
             "V540TU")
               DASHARO_REL_NAME="novacustom_v54x_mtl"
+              FLASHROM_ADD_OPT_UPDATE_OVERRIDE="--ifd -i bios"
+
```

Comment:
```suggestion
```

Reply:
We also need NEED_SMMSTORE_MIGRATION="true" for these models, please add

Reply:
done https://github.com/Dasharo/dts-scripts/commit/ae929736e0a8114e0ffc772cf689fc90818ce85a